pydantic/_internal/_config.py

```diff
@@ -199,6 +199,7 @@ def prepare_config_extra(config: ConfigDict | dict[str, Any]) -> None:
     if extra is not None and not isinstance(extra, Extra):
         try:
             config['extra'] = Extra(extra)
+            warnings.warn('`extra` should be an instance of `pydantic.Extra`, not a string', DeprecationWarning)
```

Comment:
Is there a reason we are using the `Extra` enum rather than `Literal['allow', 'forbid', 'ignore']`?

Personally, I find it pretty annoying that you have to add an import to specify this value. I recognize that currently there are various type hints indicating you need to use the `Extra` enum, and I imagine we need to keep it around for v1 compatibility, but I would prefer to reduce the use of that enum and replace it with a literal, rather than increase it.

This isn't an opinion I'd fight for though. And maybe I'm missing some other benefit to using the python Enum.

Reply:
Removed the warning for now. I'm leaving in the use of the enum for now, partially because I'm too lazy to revert it, partially because that is what the type hint says now so it's more correct, at least for now. Let me know if you want me to revert that change.